### ISSUE TITLE
Parking and flipped vehicles

### ISSUE THREAD

**Issue by codexGW** [micovery/ArmA3_Wasteland.Altis#68]
Retrieved vehicles tend to bounce around and flip a lot. This is especially bad when you have a $55,000 Kajman and it blows up when you retrieve it. As you may know helicopters that flip blow up. Maybe use a set spawn point like vehicle stores use then if those aren't there or are blocked use the logic it uses now. This will also solve the problem of vehicles being put in weird spots and having to tow them out. This can be frustrating when you don't have a vehicle that can tow out stuck retrieved vehicles.

**Comment by micovery** [comment]
Vehicle stores are purposely near large open areas in order to avoid this. If you start spawning a lot of vehicles in a vehicle store, you will find the same issue is there, where the vehicles end-up in weird places.

Since the parking locations are the ones existing from the map ... so you cannot pick a single specific spawning point (relative to the storage location)  that will work for all of them. Sometimes, the parking locations are in tight urban areas (like the one near Kavala) ... other time the parking locations is in open rural areas like (like the one in Neochori).

Also, this is only an issue if you retrieve the vehicle from a different parking location than where it was originally parked. 

If you have other alternative solutions to this issue, I'd gladly look at it. 

I know that there are ways of using script to try find a "safe" position to spawn stuff, but those only work well for players because of the small bounding box. If you try to use those on a vehicle, you end up with the vehicle clipping through buildings (if you are lucky) ... or worse exploding.  The current method uses the "createVehicle" command itself and it lets the game figure out what it thinks it's a safe location to put the vehicle within a certain radius.

**Comment by codexGW** [comment]
People use the parking a lot for transferring vehicles to locations and maps safely. I don't really play much but had problems using retrieve.  One solution maybe would be to make a option in the config where you can put the towns name then retrieve position on the map. 

pp_cities_spawnpos = ["Katalaki 12352,15764.2,0.00148201", "Agios Konstantinos 12352,15764.2,0.00148201"];

**Comment by codexGW** [comment]
Oh yeah and if the position wasn't set in the config it would use the logic it uses now. Maybe another solution would be to put no damage on the vehicle, then use the unflip, then set the damage.

**Comment by micovery** [comment]
Setting coordinates manually is not portable/scalable. Whatever the solution is, it has to be something that does not involve manually manually putting coordinates into the code, or putting markers in the editor.

**Comment by micovery** [comment]
An alternate solution would be to let the player himself pick where he wants the vehicle to spawn. Since the player himself has the knowledge of what a  "safe location" is ... he can simply walk to where he wants to spawn the vehicle (within a certain distance of the parking location).

**Comment by codexGW** [comment]
Yeah I like that a lot. Pick a spot within 200 meters or something like that would be awesome.
